src/equations/traffic_flow_lwr_1d.jl

```diff
@@ -0,0 +1,114 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    TrafficFlowLWREquations1D
+
+The classic Lighthill-Witham Richards (LWR) model for 1D traffic flow.
+```math
+\partial_t u + v_{\text{max}} \partial_1 [u (1 - u)] = 0
```

Comment:
Might be good to add information to this description that the variable u is the traffic density and that v_max is the maximum velocity (i.e. speed limit).